server_environment_iap/readme/CONFIGURE.rst

```diff
@@ -0,0 +1,14 @@
+With this module installed, the IAP Accounts are configured
+in the `server_environment_files` module (which is a module
```

Comment:
this is not mandatory, you can use `SERVER_ENV_CONFIG` + `SERVER_ENV_CONFIG_SECRET` via env vars.
I'd simply link to `server_environment` README.